src/ui/public/vislib/lib/dispatch.js

```diff
@@ -239,35 +239,11 @@ export function VislibLibDispatchProvider(Private, config) {
      * @returns {Function}
      */
     addBrushEvent(svg) {
-      if (!this.isBrushable()) return;
-
-      const self = this;
-      const xScale = this.handler.categoryAxes[0].getScale();
-      const brush = this.createBrush(xScale, svg);
-
-      function simulateClickWithBrushEnabled(d, i) {
-        if (!validBrushClick(d3.event)) return;
-
-        if (isQuantitativeScale(xScale)) {
-          const bar = d3.select(this);
-          const startX = d3.mouse(svg.node());
-          const startXInv = xScale.invert(startX[0]);
-
-          // Reset the brush value
-          brush.extent([startXInv, startXInv]);
-
-          // Magic!
-          // Need to call brush on svg to see brush when brushing
-          // while on top of bars.
-          // Need to call brush on bar to allow the click event to be registered
-          svg.call(brush);
-          bar.call(brush);
-        } else {
-          self.emit('click', self.eventResponse(d, i));
-        }
+      if (!this.isBrushable()) {
```

Comment:
do we need this check ? we already check before adding brush event.

Reply:
Yes in the we do a check before adding the brush event, but we never know if in the future we call this. Maybe I can remove the other check

Reply:
what do you mean ? if a chart is brushable its brushable .... now and in the future. 
only time this could change is when we redraw the chart, and then we re-add the events as well.

Reply:
Since we will backport that to 6.6.1 I would prefer leaving it in. We never can be 100% sure, and let's minimize the potential change.